apps/site/src/components/pages/wordpress/details.tsx

```diff
@@ -0,0 +1,530 @@
+                fontStyle: "italic",
+              }}
+            >
+              AI Blocks
+            </Typography>
+            <Box
+              sx={{
+                fontSize: "1rem",
+                mt: "20px",
+              }}
+            >
+              <Box sx={{ display: "flex", mb: "12px" }}>
+                <RightPointerIcon />
+                <Typography>
+                  The best <strong>AI text</strong> generation, style, tone,
+                  grammar and editing blocks for WordPress
+                </Typography>
+              </Box>
+              <Box sx={{ display: "flex", mb: "12px" }}>
+                <RightPointerIcon />
+                <Typography>
+                  Powerful <strong>AI image generation</strong> blocks let you
+                  illustrate posts in seconds
+                </Typography>
+              </Box>
+              <Box sx={{ display: "flex", mb: "12px" }}>
+                <RightPointerIcon />
+                <Typography>
+                  Advanced <strong>AI image editing</strong> blocks let you
+                  remove watermarks from photos, or remove/replace the
+                  backgrounds of product photos in WooCommerce
+                </Typography>
+              </Box>
+              <Box sx={{ display: "flex", mb: "12px" }}>
+                <RightPointerIcon />
+                <Typography component="div">
+                  Best-in-class <strong>AI chat</strong> block lets you persist
+                  chats, publish them on your website, branch off mid-way from
+                  prior conversations, set the AI’s response style and more
+                </Typography>
+              </Box>
+              <Box sx={{ display: "flex", mb: "12px" }}>
+                <RightPointerIcon />
+                <Typography>
+                  Switch between AI models and providers at will
+                </Typography>
+              </Box>
+              <Box sx={{ display: "flex", mb: "12px" }}>
+                <RightPointerIcon />
+                <Typography component="div">
+                  <strong>
+                    <Typography
+                      display="inline"
+                      sx={{
+                        color: ({ palette }) => palette.purple[700],
+                        fontWeight: 700,
+                      }}
+                    >
+                      Free credits
+                    </Typography>{" "}
+                    for all models and providers
+                  </strong>
+                </Typography>
+              </Box>
+            </Box>
+            <Box
+              sx={{
+                display: "flex",
+                gap: 1,
+                flexDirection: {
+                  xs: "column",
+                  md: "row",
+                },
+                mt: 1,
+              }}
+            >
+              <CustomLinkButton
+                href="/"
```

Comment:
@nonparibus, As well as the `href`. We did not want to imply the link so we're bringing it to your notice